client/verta/tests/test_versioning/test_dataset.py

```diff
@@ -332,7 +332,7 @@ def test_list_paths(self):
     def test_base_path(self, paths, base_path):
         filepaths = _file_utils.flatten_file_trees(paths)
         expected_paths = set(
-            os.path.relpath(path, base_path)
+            os.path.relpath(path, base_path).replace("\\", "/")
```

Comment:
I'm reading more about this, and it sounds bananas.

I'm pretty sure our client uses `relpath()` in a few places, and I'm concerned that this is a wider underlying problem.